recipes/avahi-mdnsresponder/system/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class AvahiMdnsResponderSystemConan(ConanFile):
+    name = "avahi-mdnsresponder"
+    provides = "mdnsresponder"
+    version = "system"
+    description = "Conan package for Avahi - Service Discovery for Linux using mDNS/DNS-SD - compatible with Bonjour"
```

Comment:
```suggestion
    description = "Avahi - Service Discovery for Linux using mDNS/DNS-SD - compatible with Bonjour"
```

Describe the library, not the conan package.